PlanGrid.Api/PlanGridHttpHandler.cs

```diff
@@ -19,13 +19,15 @@ public class PlanGridHttpHandler : HttpClientHandler
         public const HttpStatusCode RateLimitExceeded = (HttpStatusCode)429;
 
         private string authenticationToken;
+        private TokenType tokenType;
         private RefitSettings settings;
         private string version;
         private int? maxRetries;
 
-        public PlanGridHttpHandler(string accessToken, RefitSettings settings, string version, int? maxRetries)
+        public PlanGridHttpHandler(string accessToken, RefitSettings settings, string version, int? maxRetries, TokenType tokenType = TokenType.Basic)
```

Comment:
I think this will probably require some callsite changes in `PlanGridApiClient`:

https://github.com/plangrid/plangrid-api-net/blob/master/PlanGrid.Api/PlanGridClient.cs#L50

```
            var api = (AutoGeneratedIPlanGridApi)RestService.For<IPlanGridApi>(
                new HttpClient(new PlanGridHttpHandler(apiKey, settings, version, maxRetries))
                {
                    BaseAddress = new Uri(url), Timeout = TimeSpan.FromSeconds(timeout)
                },
                settings);
```

i.e. you'll want to surface this new type in the `Create` method so clients have the opportunity to override the default scheme.